_data/sfz/syntax.yml

```diff
@@ -535,6 +535,9 @@ categories:
         default: 0
         min: 0
         max: 4294967296
+      alias:
+      - name: "offby"
+        version: "SFZ v2"
```

Comment:
this seems to me more a LS extension additions, I can't see it in the book for SFZ v2, even if supported by ARIA. Correct me if I'm wrong.